src/main/java/com/microsoft/azure/kusto/kafka/connect/sink/KustoSinkTask.java

```diff
@@ -86,13 +88,14 @@ public static IngestClient createKustoIngestClient(KustoSinkConfig config) throw
                         String table = mapping.getString("table");
 
                         String format = mapping.optString("format");
+                        CompressionType compressionType = StringUtils.isBlank(mapping.optString("compression")) ? null : CompressionType.valueOf(mapping.optString("compression"));
```

Comment:
null here actually mean gzip later right? so maybe instead of null we will give it the appropriate default value?

Reply:
gz here means no need to compress later on .. null means its allready compressed - 
as i wrote before- ill change the naming so it will be clear